src/_cffi_src/openssl/ssl.py

```diff
@@ -504,6 +505,11 @@
 """
 
 CUSTOMIZATIONS = """
+// This symbol is being preserved because removing it will break users with
+// pyOpenSSL < 19.1 and pip < 20.x. We need to leave this in place until those
+// users have upgraded.
+static const long Cryptography_HAS_TLSEXT_HOSTNAME = 1;
```

Comment:
Can you add one of our deprecation symbols here in a comment, so we remember to grep for it?